Dockerfile

```diff
@@ -25,13 +25,19 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
     mv tini /usr/local/bin/tini && \
     chmod +x /usr/local/bin/tini
 
+# Install Mysql regex UDFs
+RUN wget --quiet https://github.com/mysqludf/lib_mysqludf_preg/archive/testing.zip && \
+    mkdir lib_mysqludf_preg && \
+    unzip testing.zip -d lib_mysqludf_preg && \
+    cd lib_mysqludf_preg && \
+    ./configure && make install && make installdb && make test;
```

Comment:
Seems like the "this" will work fine. Tried adding it to the docker-compose.yml (pushed to this branch), but the DB failed to start up with a long list of errors, the most important bit of which I pasted below. Any ideas on what's wrong?

db_1   | 2016-12-24 16:51:34 7fa19625e720  InnoDB: Operating system error number 13 in a file operation.
db_1   | InnoDB: The error means mysqld does not have the access rights to
db_1   | InnoDB: the directory.
db_1   | 2016-12-24 16:51:34 13 [ERROR] InnoDB: Creating or opening ./ibdata1 failed!
db_1   | 2016-12-24 16:51:34 13 [ERROR] InnoDB: Could not open or create the system tablespace. If you tried to add new data files to the system tablespace, and it failed here, you should now edit innodb_data_file_path in my.cnf back to what it was, and remove the new ibdata files InnoDB created in this failed attempt. InnoDB only wrote those files full of zeros, but did not yet use them in any way. But be careful: do not remove old data files which contain your precious data!